packages/customize-widgets/src/index.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * WordPress dependencies
+ */
+import { registerBlockType } from '@wordpress/blocks';
+import {
+	registerCoreBlocks,
+	__experimentalGetCoreBlocks,
+	__experimentalRegisterExperimentalCoreBlocks,
+} from '@wordpress/block-library';
+
+/**
+ * Internal dependencies
+ */
+import CustomizeControl from './customize-control';
+
+const { wp } = window;
+
+/**
+ * Initializes the widgets block editor in the customizer.
+ */
+export function initialize() {
```

Comment:
We'll eventually want to DRY a lot of this up too. Some of it already exists in `edit-widgets`. Not right now though.